apps/ensindexer/src/lib/graphnode-helpers.test.ts

```diff
@@ -2,14 +2,14 @@ import { beforeEach, describe, expect, it, vi } from "vitest";
 
 import type { LabelHash } from "@ensnode/ensnode-sdk";
 
-import { setupConfigMock } from "./utils/mockConfig";
+import { setupConfigMock } from "@/lib/__test__/mockConfig";
```

Comment:
i've moved test-related meta files to `@/lib/__test__` which seems to be relatively well-established pattern